### DISCUSSION TITLE
移动端的显示纹理

### DISCUSSION THREAD

**Discussion by HHaidai** [guansss/pixi-live2d-display#147] [General]
<img width="425" alt="企业微信截图_17140284496385" src="https://github.com/guansss/pixi-live2d-display/assets/115163343/c7e3dadd-13d7-4930-86ee-feb074dcd50d">
左侧是该库生成的模型，右侧是官方demo生成的模型，能感觉出来左侧稍微模糊了点，个人推测是模型纹理的问题，期待作者早日修复

**Comment by HHaidai** [comment]
经过一天的研究，发现是缩放的原因，index_html中的meta标签的width如果是device-width就会模糊，因为手机上边宽度差不多只有400左右，如果把width的数值扩大到2000左右就会非常清晰，官方demo也是拓展到了1900才看起来清晰的，这个有办法不修改meta解决么？

**Comment by mydaoyuan** [comment]
我是修改了官方代码中的  _resizeCanvas 函数

```
  private _resizeCanvas(): void {
    canvas.width = window.innerWidth * window.devicePixelRatio;
    canvas.height = window.innerHeight * window.devicePixelRatio;
  }
```
增加了 * window.devicePixelRatio
